src/__mocks__/testResults.ts

```diff
@@ -93,11 +93,11 @@ export const mockTestResultResponse: any = {
 
 export const mockPanelTestResult = {
   data: {
-    uuid: '161430AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+    uuid: '5b0cdc41-7371-4c2f-a1e4-ed1bf7416a8d',
```

Comment:
Is there a reason of changing mock test to Anaemia Panel? since the mock data already is a panel

Reply:
Routine panel doesn't have set members thats why we are using anaemia panel

Reply:
we could have just added setMembers in that case.. but that's fine. Just thought if a mock is updated with different data all the tests which uses that mock and have check on any value of in the mock need to be updated..